core-bundle/src/Image/ImageFactory.php

```diff
@@ -202,9 +202,12 @@ private function createConfig(array|int|null $size, ImageInterface $image): arra
                     $this->enhanceResizeConfig($config, $imageSize->row());
                     $options->setSkipIfDimensionsMatch((bool) $imageSize->skipIfDimensionsMatch);
 
-                    if (!$imageSize->preserveMetadata) {
+                    if ('metaDelete' === $imageSize->preserveMetadata) {
                         $options->setPreserveCopyrightMetadata([]);
-                    } elseif ($preserveMetadata = StringUtil::deserialize($imageSize->metadata, true)) {
+                    } elseif (
+                        'metaOverwrite' === $imageSize->preserveMetadata
+                        && ($preserveMetadata = StringUtil::deserialize($imageSize->metadata, true))
+                    ) {
```

Comment:
@ausi Am I right that we do not have to handle the `metaDefault` case here, because it is already handled on line 194?

Reply:
Correct.